### DISCUSSION TITLE
Port Forwarding

### DISCUSSION THREAD

**Discussion by vhdsoccer** [GerryDazoo/Slinger#89] [Q&A] [answered]
I'm one one side of the ocean, my Slingbox on the other side, at my father's home.

I edited four lines on the config id : ipadress, port (both found after connecting on newwatch), password and finderid (following the instructions here).

When trying to connect, I receive the following error lines : 
Error conntecting to ('ip', port)
No valid slingbox ip info found in config.ini

After reading conversations here, it looks like port forwarding on the router is needed.

First question : on which router ? Mine or the one of my father ?

I'm a newbie with routers (and my father does not even now how to access this one, but I could help him...), so I don't know which information to enter.

When I try mine, here are the fiels I must fill:

Service port
Internal port (not mandatory)
IP address
Standard service port (I have a long list, including among other dns, http, smtp, telnet,...)

Service port: is that the port of the slingbox shown in newwatch (5201) ?
IP address: is that the personal IP address shown in newwatch, or something else ?
Also, standard service port, is that http or something else ?

Thank you for your help !
And also thank you Gerry for this wonderful project !

**Comment by GerryDazoo** [comment, marked as answer]
The port forwarding rule need to be on your father's router. Yes forward port 5201 to the internal IP address of the slingbox.

**Comment by f520k510** [comment]
You need to configure port forward on your father's router.
forward tcp port 5201 to the slingbox's internal ip.

**Comment by vhdsoccer** [comment]
Great, apparently it connects now, the error messages have disappeared.

Yet, it does not work yet, now there is another hurdle later.

Getting external IP address
Redirector, my external IP address xx.xx.xx.xx (that's the IP of my computer here at home)
PING failure ('xx.xx.xx.xx', 8080) [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
No working external portmaps. Not registering slingboxes

**Reply by GerryDazoo** [reply]
It's working. Just send a streaming request. Comment out your ;finderid=   line. until you decide you really need it. See the release notes. This will get rid of those messages.

**Comment by vhdsoccer** [comment]
Great, it works ! Thank you !
I even was able to use it with Emby on my TV :)
Next step, figure how to change the channels without needing to sit in front of my computer.

**Reply by GerryDazoo** [reply]
I use my phone as a remote.

**Reply by vhdsoccer** [reply]
Great, I found also the way to connect my cell to my localhost and to use it as a remote.
In the meantime, the Emby connexion does not work anymore (it always stops), thus I use VLC on my computer and use the wireless connection to send it to my TV.
I hope that someday I'll be able to watch my slingbox directly from my TV and browse the channels there easily, but already being able to use it after November 9 is already awesome !
Thanks again, Gerry
